### PR TITLE
Bones II: The Recrackening

### PR DESCRIPTION
**Pull request by Megamaw**
This change lowers the speed of skeletons.

I can't imagine anyone has any problems with this.

EDIT: Removed part of PR that removes 'ACK ACK!' as a racial quirk. Apparently people REALLY like it for some reason.

### CODE REVIEW

**Comment by nuklearcellphoneg** [comment]
wha

**Comment by TheFriggenSawbones** [comment]
why

**Comment by ghost** [comment]
fuck off no skeletons allowed

**Comment by kilozombie** [comment]
ack?

ack... 



 ack ack  , ack . ack ack, ack. ! ack ack ! ack ack, ack ack ack. ack ack. ack ack, ack! ack? ack ack ack. 
 ack ack  , ack . ack ack, ack. ! ack ack ! ack ack, ack ack ack. ack ack. ack ack, ack! ack? ack ack ack. 
 ack ack  , ack . ack ack, ack. ! ack ack ! ack ack, ack ack ack. ack ack. ack ack, ack! ack? ack ack ack. 
 ack ack  , ack . ack ack, ack. ! ack ack ! ack ack, ack ack ack. ack ack. ack ack, ack! ack? ack ack ack. 
 ack ack  , ack . ack ack, ack. ! ack ack ! ack ack, ack ack ack. ack ack. ack ack, ack! ack? ack ack ack. 
 ack ack  , ack . ack ack, ack. ! ack ack ! ack ack, ack ack ack. ack ack. ack ack, ack! ack? ack ack ack. 
 ack ack  , ack . ack ack, ack. ! ack ack ! ack ack, ack ack ack. ack ack. ack ack, ack! ack? ack ack ack. 
 ack ack  , ack . ack ack, ack. ! ack ack ! ack ack, ack ack ack. ack ack. ack ack, ack! ack? ack ack ack. 
 ack ack  , ack . ack ack, ack. ! ack ack ! ack ack, ack ack ack. ack ack. ack ack, ack! ack? ack ack ack. 
 ack ack  , ack . ack ack, ack. ! ack ack ! ack ack, ack ack ack. ack ack. ack ack, ack! ack? ack ack ack. 
 ack ack  , ack . ack ack, ack. ! ack ack ! ack ack, ack ack ack. ack ack. ack ack, ack! ack? ack ack ack.

**Comment by jakmak3** [comment]
is this what i look like when i talk about robots?

**Comment by Diavill** [comment]
i came here just to laugh at you

**Comment by Megamaw** [comment]
removed the part that removes ack ack.

i have literally heard no one, ever, defend it until now.

**Comment by kilozombie** [comment]
The ack is like the entire reason that skellingtons exist
Also why does this PR exist, please explain it

**Comment by nuklearcellphoneg** [comment]
It's like you're taking a step forward at the same time as a step back and you're doing a split.

Careful, that could

crack a bone

**Comment by Exxion** [comment]
You heard people defend it last time you tried to remove it

**Comment by Intigracy** [comment]
https://github.com/vgstation-coders/vgstation13/issues/11287
https://github.com/vgstation-coders/vgstation13/issues/11287
https://github.com/vgstation-coders/vgstation13/issues/11287
https://github.com/vgstation-coders/vgstation13/issues/11287
https://github.com/vgstation-coders/vgstation13/issues/11287
https://github.com/vgstation-coders/vgstation13/issues/11287
https://github.com/vgstation-coders/vgstation13/issues/11287

**Comment by Megamaw** [comment]
i will say right now exxion two things
1. i have poor memory retention and
2. i am not a clever man

**Comment by Riekland** [comment]
can someone say something stupid so I can get mad at them and turn this into my own personal blog about how I hate skeletons please

**Comment by SonixApache** [comment]
the obvious solution is to remove megamaw

File: code/modules/mob/living/carbon/species.dm [hunk 1]
```diff
@@ -337,12 +337,12 @@ var/global/list/whitelisted_species = list("Human")
...
 		)
 
-	move_speed_multiplier = 1.5
+	move_speed_multiplier = 1.2
 
 	primitive = /mob/living/carbon/monkey/skellington
 
 /datum/species/skellington/handle_speech(var/datum/speech/speech, mob/living/carbon/human/H)
-	if (prob(25))
+	if (prob(20))
```
Comment (SonixApache) [inline code/modules/mob/living/carbon/species.dm:345]: c'mon son

**Comment by Chazach** [comment]
repo ban when

File: code/modules/mob/living/carbon/species.dm (hunk 1 above)
Comment (ghost) [inline code/modules/mob/living/carbon/species.dm:345]: absolutely disgusting

**Comment by ghost** [comment]
Don't do a dylan and hide things you're not saying you're doing Ack Ack

**Comment by Megamaw** [comment]
oh fuck lmao that was the wrong number

**Pull request closed without merging**